src/sqltest/parts/dashboard/widgets/propertiesWidget.component.test.ts

```diff
@@ -58,6 +58,7 @@ suite('Dashboard Properties Widget Tests', () => {
 			serverEdition: undefined,
 			azureVersion: undefined,
 			osVersion: undefined,
+			options: { isBigDataCluster: false, clusterEndpoints: [] },
```

Comment:
Again, do we need to special case these? I don't think these should be hardcoded, we wanted to move away from that.